<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="Textbox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>inputfile name:</string>
    </property>
    <widget class="QPushButton" name="inputPushButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QRadioButton" name="SimpleAlgorithm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>SimpleAlgorithm</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="DelaunayAlgorithm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>DelaunayAlgorithm</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="runButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>RUN</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_by">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2014-9      by: victor He</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionAddPoint"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>518</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="Menu_Option">
    <property name="title">
     <string>Start</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
    <addaction name="actionRun"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
   </widget>
   <addaction name="Menu_Option"/>
   <addaction name="menuOption"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionAddPoint">
   <property name="text">
    <string>AddPoint</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>inputPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
